demo/demo.md

```diff
@@ -248,119 +327,114 @@ While Auro components, `auro-alert` included, come with a pre-defined UI opinion
+
+```html
+<auro-alert fixed type="warning">This is warning message</auro-alert>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+## a11y alerts
+
 ### Hidden visually and from screen readers
 
 <div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/hidden.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/hidden.html -->
   <auro-alert hidden type="error">This content will be hidden visually and from screen readers.</auro-alert>
-</div>
-
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
 <auro-accordion alignRight>
-  <span slot="trigger">See code</span>
-
-  ```html
-  <auro-alert hidden type="error">This content will be hidden visually and from screen readers.</auro-alert>
-  ```
-</auro-accordion>
-
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/hidden.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/hidden.html -->
+
+```html
+<auro-alert hidden type="error">This content will be hidden visually and from screen readers.</auro-alert>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
 ### Hidden visually
 
 <div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/hidden-visually.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/hidden-visually.html -->
   <auro-alert hiddenVisually type="error">This content will be hidden visually, but screen readers will still pick it up.</auro-alert>
-</div>
-
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
 <auro-accordion alignRight>
-  <span slot="trigger">See code</span>
-
-  ```html
-  <auro-alert hiddenVisually type="error">This content will be hidden visually, but screen readers will still pick it up.</auro-alert>
-  ```
-</auro-accordion>
-
-
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/hidden-visually.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/hidden-visually.html -->
+
+```html
+<auro-alert hiddenVisually type="error">This content will be hidden visually, but screen readers will still pick it up.</auro-alert>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
 ### Hidden from screen readers
 
 <div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/hidden-audible.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/hidden-audible.html -->
   <auro-alert hiddenAudible type="error">This content will be hidden from screen readers.</auro-alert>
-</div>
-
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
 <auro-accordion alignRight>
-  <span slot="trigger">See code</span>
-
-  ```html
-  <auro-alert hiddenAudible type="error">This content will be hidden from screen readers.</auro-alert>
-  ```
-</auro-accordion>
-
-
-## Recommended Use and Version Control
```

Comment:
Re-insert this example back in and create a `custom.html` file to be used for the example. See here:
- https://github.com/AlaskaAirlines/auro-input/blob/6c805fa0c571f78d3786832679f9d70780533e50/apiExamples/custom.html
- https://github.com/AlaskaAirlines/auro-input/blob/6c805fa0c571f78d3786832679f9d70780533e50/docs/partials/index.md?plain=1#L144-168